Add UnPoller to Templates

Griffen8280 opened the pull request:
<!-- The title of the pull request should be a short description of what was done.  -->

<!-- You can remove any parts of this template not applicable to your Pull Request.  -->

# Summary

Created all necessary files and documenation to show the process of installing 2 new containers and attaching them to the Raspberry Pi Monitoring grafana instance.

# Why This Is Needed

Will provide a dashboard for users running the Unifi Controller to better see the statistics of their network.

# What Changed

Updating of the template file while adding the logos and installer script needed to integrate the project.

## Added

UnPoller container information as well as configuration files

## Changed

N/A

## Fixed

No

## Removed

No

# Screenshots

![USG-UnPoller](https://user-images.githubusercontent.com/42878642/139708070-dc972f3c-0833-46d2-b1ac-1d56883d6edc.PNG)

![UAP-unpoller](https://user-images.githubusercontent.com/42878642/139708088-82d2921c-dd66-43c8-bfed-4fb08a105e1e.PNG)

# Checklist

<!-- You can erase any parts of this template not applicable to your Pull Request. -->
- [x] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?
- [x] Does your submission pass tests?
- [x] Have you linted your code locally prior to submission?
- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
- [x] Have you updated documentation, as applicable?

rmiddle reviewed (changes requested):
Static IP defined in config needs to be changed to hostname instead.

Griffen8280 commented:
> Static IP defined in config needs to be changed to hostname instead.

Fixed.  Updated the compose file to only create a netowrk for the contianers.  The config file has been updated to use the hostname instead of an IP.
